cbmc/invntt_layer321/invntt_layer321_harness.c

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 The mlkem-native project authors
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+/*
+ * Insert copyright notice
+ */
+
+/**
+ * @file invntt_layer321_harness.c
+ * @brief Implements the proof harness for invntt_layer321 function.
+ */
+
+/*
+ * Insert project header files that
+ *   - include the declaration of the function
+ *   - include the types needed to declare function arguments
```

Comment:
please remove these boilerplate comments as done in https://github.com/pq-code-package/mlkem-native/pull/518